ociclient/client.go

```diff
@@ -552,3 +565,215 @@ func AddKnownMediaTypesToCtx(ctx context.Context, mediaTypes []string) context.C
 	}
 	return ctx
 }
+
+// *************************************************************************************
+// Docker Manifest v2 Schema 1 Support
+// see also:
+// - https://docs.docker.com/registry/spec/manifest-v2-1/
+// - https://github.com/moby/moby/blob/master/image/v1/imagev1.go
+// - https://github.com/containerd/containerd/blob/main/remotes/docker/schema1/converter.go
+// *************************************************************************************
+
+const (
+	DockerV2Schema1MediaType       = "application/vnd.docker.distribution.manifest.v1+json"
+	DockerV2Schema1SignedMediaType = "application/vnd.docker.distribution.manifest.v1+prettyjws"
+	MediaTypeImageLayerZstd        = "application/vnd.oci.image.layer.v1.tar+zstd"
```

Comment:
arent these conststants part of the containerd lib?
so we should not duplicate them

e.g. https://github.com/containerd/containerd/blob/f0a32c66dad1e9de716c9960af806105d691cd78/remotes/docker/schema1/converter.go#L81

Reply:
Ye you're right, I changed DockerV2Schema1SignedMediaType to be used from the containerd definition. The other two constants aren't defined in any of our depedencies. Maybe move them to constants.go?

Reply:
> Another thing: should I move them to constants.go?
no these constants are only used here so no need to have them in the shared constants file